intercept/leave.js

```diff
@@ -1,7 +1,9 @@
 const core = require('@actions/core')
 const exec = require('@actions/exec');
+const MetritonClient = require('../src/metrics')
 
 const telepresenceLeave = async function(){
+    MetritonClient.sendMetricsReport('leave')
```

Comment:
I'd push this line after the `telepresence_service_intercepted` where we actually run `telepresence leave`